pinot-core/src/main/java/org/apache/pinot/core/segment/store/ColumnIndexType.java

```diff
@@ -23,7 +23,8 @@
   FORWARD_INDEX("forward_index"),
   INVERTED_INDEX("inverted_index"),
   BLOOM_FILTER("bloom_filter"),
-  NULLVALUE_VECTOR("nullvalue_vector");
+  NULLVALUE_VECTOR("nullvalue_vector"),
+  TEXT_INDEX("text_index");
```

Comment:
I am not sure whether this is persisted or not. If it is, then we may need to rename this also to include lucene

Reply:
This is non-persistent